server/websocket.go

```diff
@@ -48,6 +48,10 @@ const (
 	wsReconnectionTimeout            = 10 * time.Second
 )
 
+var (
+	sessionAuthCheckInterval = 10 * time.Second
```

Comment:
@esarafianou For performance reasons I wouldn't go lower than this, let me know if it's reasonable.

Reply:
Totally reasonable.